### ISSUE TITLE
Map edge shroud borders are extremely annoying

### ISSUE THREAD

**Issue by reaperrr** [OpenRA/OpenRA#11569]
While they might be more appealing from an aesthetic point of view, in D2k you can currently hide vehicles almost completely(!) below the "shroud" of map borders, and it's not that much better in the other mods, either. This has been bothering me for ages, I think it's long overdue we adressed this.

I think we should a) allow modders to disable the edge hiding in their mods, and b) fix it as far as possible in our shipping mods.

If we don't want to change the underlying logic, we should at least use custom shroud art offsets for map borders to reduce the actor-hiding as much as possible.

**Comment by pchote** [comment]
The longstanding plan has been to introduce a several-cell decorative border around the edge of the map that you can see but not order units into.

**Comment by obrakmann** [comment]
Could we, as a first step, draw the border one cell further out?

**Comment by obrakmann** [comment]
Could we, as a first step, draw the border one cell further out?

**Comment by reaperrr** [comment]
> The longstanding plan has been to introduce a several-cell decorative border around the edge of the map that you can see but not order units into.

That would require updating all (or at least most) map's `.bin` files though, since Westwood's original maps were not designed for that.

**Comment by pchote** [comment]
Indeed.  I imagine we would have a flag to enable/disable it per-map so that we can fixup / migrate maps over time.

**Comment by reaperrr** [comment]
How about making the "thickness" in cells of the decorative border customizable instead, and defaulting to 0 to keep backwards-compatibility?

**Comment by pchote** [comment]
That would work well.

**Comment by CH4Code** [comment]
How about painting everything non-neutral on top of border shroud instead of shrounding the units? (just spent 5 minutes in a missionsearching for remaining units only to find the last harvester in the border shroud by chance :/ )
Would nicely fix out of bounds aircraft as well and only look a bit weird on the lower edge.

**Comment by MikillRosen** [comment]
bit of a necropost, but why not just remove the map edge shroud borders? I don't get the problem
